data/hai/hai news.txt

```diff
@@ -0,0 +1,251 @@
+# Copyright (c) 2020 by Nomadic Volcano
+# Based on earlier text copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# Visual suggestion: a bit quirky or weird; expression is content, confident, or stoic
+news "food in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Visiting chef"
+			"Food connoisseur"
+			"Tourist"
+			"Tour guide"
+	portrait
+		portrait/human26
+		portrait/human31
+		portrait/human57
+		portrait/human68
+	message
+		word
+			"This may sound odd, but the acorn cookies they sell here are the best cookies I've ever eaten."
+			"A Hai-Human fusion restaurant near here sells peanut pizza. It sounds gross, but the taste is heavenly."
+			"They make beer from fermented nuts here. I didn't know you could do that!"
+			"I just can't get enough acorn stew. These Hai cooks are amazing."
+			"Peanut this, walnut that, hazelnut icecream. What I wouldn't give for some good steak right now..."
+
+
+# Visual suggestion: glasses, clean-cut attire, possibly scientific accoutrements
+news "trading in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Merchant captain"
+	portrait
+		portrait/human02
+		portrait/human04
+		portrait/human06
+		portrait/human10
+		portrait/human15
+		portrait/human17
+		portrait/human31
+		portrait/human58
+		portrait/human63
+		portrait/human72
+		portrait/human73
+		portrait/human76
+		portrait/human79
+		portrait/human98
+		portrait/human108
+		portrait/human112
+		portrait/human125
+	message
+		word
+			"Hai Tech is incredible compared to ours. Every time I add a ship to my fleet, I always make a trip up here to stock up on Hai tech."
+			"Have you tried out one of those Hai anti-missile systems? They'll waste any missile in Human space."
+			"Can you believe these Hai? They're light-years ahead of us, right under our noses."
+			"Have you visited the Quarg ringworld here in Hai space? It's a humbling sight. There's probably more Quarg in there than there are humans anywhere. Makes you feel pretty insignificant."
+			"Did you see the stuff they sell on Icelake? Hai technology is amazing."
+			"I met some young Hai pilots. It sounds like they start out as poor as we do."
+			"I sold my entire stock of food at this port. I guess even Hai get tired of nuts."
```

Comment:
The implication here is that it's non-Hai food, so maybe the merchant should explicitly state that.
```suggestion
			"I sold my entire stock of food from human space at this port. I guess even Hai get tired of nuts."
```